src/textual/widgets/_data_table.py

```diff
@@ -309,6 +314,8 @@ class DataTable(ScrollView, Generic[CellType], can_focus=True):
     show_cursor = Reactive(True)
     cursor_type: Reactive[CursorType] = Reactive[CursorType]("cell")
     """The type of the cursor of the `DataTable`."""
+    cell_padding = Reactive(_DEFAULT_CELL_X_PADDING)
```

Comment:
Will this re-render the table if set after the first render?

Reply:
It does.

Reply:
Are you absolutely certain of this? Like if you were John McClane and Hans Gruber was pointing a gun to your head, and a wrong answer would lead to Nakatomi Plaza blowing up and killing all the hostages. Would this still be your answer?

Reply:
I don't want to blow Nakatomi Plaza up. I had tested assigning to `cell_padding` while an app was running and the data table refreshed correctly.
However, I tried hooking up an `Input` to the cell padding and if I make more than one modification, it wasn't.

This was tested and fixed in bbe5c85.

Reply:
Also, I don't get Die Hard references.

Teletubbies references are more likely the way to go.

![giphy](https://github.com/Textualize/textual/assets/5621605/02418293-a3f4-431e-a2e5-eba4f71b612d)

Reply:
Thanks for your input! See 1936f10.